taccsite_cms/templates/djangocms_blog/includes/blog_meta.html

```diff
@@ -1,37 +1,43 @@
+{# https://github.com/nephila/djangocms-blog/blob/1.1.1/djangocms_blog/templates/djangocms_blog/includes/blog_meta.html #}
```

Comment:
**Note.** TACC's overwrite of this template is not identical to the diff below. The diff below is between (1) the markup I guessed-at-and-committed years ago (but never actually used) and (2) the markup changes I must make now to support the design. See the template at the URL above if you want to review the actual overwrite. Sorry.